newIDE/app/public/external/piskel/piskel-main.js

```diff
@@ -26,15 +26,15 @@ const closeWindow = () => {
 
 // Repeatedly try to gain access to piskel's control element and its methods
 // When succeeding, stop trying.
-let pskl = null;
+let pskl = document.querySelector('#piskel-frame').contentWindow.pskl;
 const tryToGetPiskel = () => {
   pskl = document.querySelector('#piskel-frame').contentWindow.pskl;
-  if (pskl !== null) {
+  if (typeof pskl === 'object') {
```

Comment:
@blurymind This is the line that fixed the issue (as far as I can tell). See my more detailed comment.

As we know that pskl is either undefined or an object, you can also do `if (pskl !== undefined)` (testing for object works but is in theory a bit too precise).